lib.py

```diff
@@ -325,8 +325,40 @@ def pay(self, amount_msat: int, binding = None):
 
             if payment is not None:
                 results[m.id] = payment
-            else:
-                results[m.id] = None
+
```

Comment:
This is where I update the outlays after a payment is made. I think maybe it can be improved a bit.

Reply:
Yea this makes sense in case something goes wrong. Was just trying to make less db calls, but this is worth it

Reply:
Yeah, so when I had the original error related to negative Millisatoshi, we would pay 4 members, then the exception would occur on the 5th. Those prior outlays weren't getting updated. So had to fix this.